CROSS_COMMUNITY_COUNCIL.md

```diff
@@ -0,0 +1,11 @@
+# Cross Community Council (C3)
+
+In this document we will collect requests, recommendations, and requirements for the Cross Community Council's charter.
+
+Charters are legal documents, and we should not be writing legalese! (Can you say: liability?) The Node.js Foundation and the JS Foundation will provide legal support to draft the official document. We will have ample chance to review, ask questions, revise, and refine this document over the next few months.
```

Comment:
I have a different perspective here.  As nothing will be adopted by the foundation (either, both, or merged) without legal review, the liability comment is a red herring here.  I have found that people not taking the time to understand what goes into various legal documents inevitably leads to misunderstandings and often frustration and wasted work.  Statements that were made in the Governance session about the need to be able to rapidly evolve bylaws, and a number of suggestions as to what should be in the bylaws, are examples of this for me.

https://github.com/nodejs/bootstrap/issues/8 is a start at a top-down understanding of what a merged foundation will look like.  This ideally should dovetail with a number of bottoms-up initiatives that have starte.

Reply:
Updated with improved language, let me know what you think 👍